cpp/src/hash/managed_allocator.cuh

```diff
@@ -20,30 +20,24 @@
 #include <new>
 
 #include <rmm/rmm.h>

```

Comment:
What about the non-managed allocator? We don't use the managed allocator in cuDF anymore.

Reply:
Should we consider renaming the file to something like `hash_map_allocator.cuh` or something like this?

Reply:
Also, do we want to add other device_memory_resource here?

Reply:
Renaming is a good suggestion.